src/agent.cpp

```diff
@@ -799,7 +814,9 @@ Agent::doReorient()
     if (! ( ball.rposValid() && ball.velValid() )) {
       dlog.addText( Logger::TEAM,
 		    __FILE__": search ball" );
-      return Bhv_NeckBodyToBall().execute( this );
+
+      Bhv_NeckBodyToBall().execute( this );
```

Comment:
Bhv_NeckBodyToBall always returns true, so lets go back to:

return Bhv_NeckBodyToBall().execute( this );